### ISSUE TITLE
hw08 ready for grading

### ISSUE THREAD

**Issue by Kozp** [Kozp/STAT545-hw-Kozik-Pavel#8]
[Here is a link to the created app](https://kozp.shinyapps.io/hw08/)
[Here is a link to the homework directory](https://github.com/Kozp/STAT545-hw-Kozik-Pavel/tree/Side-Branch/hw08%20-%20App)
[Here is a link to the user interface](https://github.com/Kozp/STAT545-hw-Kozik-Pavel/blob/Side-Branch/hw08%20-%20App/ui.R) and [server](https://github.com/Kozp/STAT545-hw-Kozik-Pavel/blob/Side-Branch/hw08%20-%20App/server.R)

**Comment by sepkamal** [comment]
Hi again @Kozp 

Your graphs look amazing! Clearly the master of `ggplot2`. 

I like that you chose a dataset other than the the BC liquors. The filtering by year works great. I especially like the "play" button! It looks like that was achieved with the ` animate = TRUE` option.

The donwload button is functional and provides the filtered data. I also appreciate your comments in your server code, as it would have been very hard to follow otherwise.

If you wanted to improve this further, I would suggest playing around with the design. For example a large, colorful title, different fonts, separate tabs for the graph and table, etc. This could be done using shinyjs and CSS. I also found `shinythemes` to be a very easy way to spice things up. All that is required is the line theme = shinytheme("journal") in the UI.

You did a good job explaining to the user what your app does. You also did a good job citing your data.

I also experienced errors when publishing to the web that did not appear in RStudio. I found viewing the website's logs in my shinyapps.io account to be helpful for this purpose.

Nice App!

Cheers,
Sep

**Comment by menglinzhou** [comment]
Hi, @Kozp!
I'm very happy to review your assignment.
- You choose another dataset other than BC Liquor, which is really fantastic! And you provide the source of your raw data, which is really helpful.
- Your app is deployed online on shinyapps.io, and it could work. And your graphs look amazing! 
- All the buttons in your app are functional. I am impressed with the play bottom beside the select bar of year, which is really interesting!
- The comments in your server code is really nice.
- To improve you app, I think you can add more selecting bottoms in your slider bar and make it possible for us to filter with other variables.
- Meanwhile, the format of your app can be adjusted. The text in `mainPanel` can be larger and the plots and tables can be just on the right of the page, which is the same like the BC App in class.

Nice job!
Menglin
